lesson4/Lesson4HW/app/src/main/java/com/margaret/lesson4hw/SettingsFragment.java

```diff
@@ -42,21 +41,11 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
 
         return view;
     }
-    public void onCreate() {
-        SharedPreferences sharedPref = getActivity().getPreferences(Context.MODE_PRIVATE);
-
-        int defaultValue = getResources().getColor(R.color.white);
-        int background = sharedPref.getInt(MainActivity.SAVED_COLOR, defaultValue);
-
-
-        getView().setBackgroundColor(background);
-    }
```

Comment:
Get rid of functions you aren't using!  Android Studio will give you a grey underline for things you aren't using, get rid of them before you submit your final code.